database.py

```diff
@@ -0,0 +1,79 @@
+import peewee
+
+class Result:
+    def __init__(self, is_success=True, data=None):
+        self.is_success = is_success
+        self.data = data
+
+class Database:
+    def __init__(self, db_name, uname="test", pwd="test", host="localhost"):
+        self.db = peewee.PostgresqlDatabase(
+            db_name,
+            user=uname,
+            password=pwd,
+            host=host
+        )
+
+        # This is taken mostly from the Peewee sample app
+        class BaseModel(peewee.Model):
+            class Meta:
+                database = self.db
+        
+        class Resume(BaseModel):
+            user_id = peewee.IntegerField(primary_key=True)
+            resume = peewee.TextField()
+            id = peewee.AutoField()
+
+        self.Resume = Resume
+
+    def start_connection(self):
+        self.db.connect()
+        self.db.create_tables([self.Resume])
+        self.db.commit()
+
+    def end_connection(self):
+        self.db.close()
+
+    def push_resume(self, user_id, resume):
+        try:
+            q = self.Resume(user_id=user_id, resume=resume)
+            n_rows_modified = q.save()
```

Comment:
What's being modified here?  Just inserting a resume right?

Reply:
Yep. There should only be a single row added, but I'm checking it in case it returns something else for whatever reason.